server/tests/factory/user-factory.js

```diff
@@ -26,6 +26,22 @@ const createTestUser = async ({
   return newUser;
 };
 
+const createNonActiveUser = async ({
```

Comment:
If you can, how about you add a ```active: false``` to ```testUserNoArgumentPassed``` .
I think this function was created to simulate a non-active user. So we do not duplicate functions.

Otherwise, looks great to me. Good job.